zipkin-storage/cassandra-v1/src/main/java/zipkin2/storage/cassandra/v1/SelectRemoteServiceNames.java

```diff
@@ -17,33 +17,35 @@
 import com.datastax.driver.core.ResultSet;
 import com.datastax.driver.core.ResultSetFuture;
 import com.datastax.driver.core.Session;
-import com.datastax.driver.core.querybuilder.QueryBuilder;
 import java.util.List;
 import java.util.Locale;
 import zipkin2.Call;
 import zipkin2.storage.cassandra.internal.call.DistinctSortedStrings;
 import zipkin2.storage.cassandra.internal.call.ResultSetFutureCall;
 
+import static com.datastax.driver.core.querybuilder.QueryBuilder.bindMarker;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.eq;
+import static com.datastax.driver.core.querybuilder.QueryBuilder.select;
+import static zipkin2.storage.cassandra.v1.Tables.REMOTE_SERVICE_NAMES;
+
 final class SelectRemoteServiceNames extends ResultSetFutureCall<ResultSet> {
 
-  static class Factory {
+  static final class Factory {
     final Session session;
     final PreparedStatement preparedStatement;
-    final DistinctSortedStrings remoteServiceNames =
-      new DistinctSortedStrings("remote_service_name");
 
     Factory(Session session) {
       this.session = session;
-      this.preparedStatement = session.prepare(QueryBuilder.select("remote_service_name")
-        .from(Tables.REMOTE_SERVICE_NAMES)
-        .where(QueryBuilder.eq("service_name", QueryBuilder.bindMarker("service_name")))
-        .limit(QueryBuilder.bindMarker("limit_")));
+      this.preparedStatement =
+        session.prepare(select("remote_service_name").from(REMOTE_SERVICE_NAMES)
+          .where(eq("service_name", bindMarker()))
+          .limit(1000));
```

Comment:
you'll notice constants like this around. we had assigned them via variables before, this is just a bit more obvious.